### PR TITLE
MGMT-14989: Document how to fetch JIRA_ACCESS_TOKEN for triage search users

### PR DESCRIPTION
**Pull request by nmagnezi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,24 +31,33 @@ There is now a facility to deep search for issues matching a content regex and a
 
 ### Examples
 
+#### Generate a Jira access token
+
+1. Make sure you are connected to Red Hat VPN.
```
Comment (javipolo) [inline README.md:36]: AFAIK VPN is not needed to access jira, so you can get the PAT without being connected

**Review by javipolo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nmagnezi) [reply README.md]: Done. 
I've added this line above and clarified that it is needed to download the logs

**Review by javipolo — APPROVED** [review]

**Pull request merged**